src/com/amazon/ion/benchmark/WriteRandomIonValues.java

```diff
@@ -277,7 +277,7 @@ private static BigDecimal randomSecondWithFraction(Random random, int scale) {
     public static void writeRandomTimestamps(int size, IonType type, String path, String timestampTemplate, String format) throws Exception {
         File file = new File(path);
         try (IonWriter writer = WriteRandomIonValues.formatWriter(format, file)) {
-           WriteRandomIonValues.writeRequestedSizeFile(size, writer, file, type, 0, null, null, null, timestampTemplate, null, null);
+           WriteRandomIonValues.writeRequestedSizeFile(size, writer, file, type, 0, null, null, null, timestampTemplate, null, null, null);
```

Comment:
Here's a suggestion for how to improve the readability of these method calls that contain tons of nulls. Consider creating helper methods for each type that call through to `WriteRandomIonValues.writeRequestedSizeFile`. The helper method fills in the nulls for the options that aren't relevant for that type.

Reply:
> Here's a suggestion for how to improve the readability of these method calls that contain tons of nulls. Consider creating helper methods for each type that call through to `WriteRandomIonValues.writeRequestedSizeFile`. The helper method fills in the nulls for the options that aren't relevant for that type.

Inside of the helper method, there will be a process to assign unused parameters to null/0, instead of writing helper methods for each type, how about defining class variables which with default value 'null'? In this way if some specific variables are not used, they will be assigned as null automatically and will avoid writing the same functionalities. Changes will be updated in the following commit, if there's better idea, please feel free to leave comment.

Reply:
This is better. I've added a small suggestion related to this on the new revision.